### PR TITLE
Corrected typo in index.js file

### PR DESCRIPTION
**Pull request by 96RadhikaJadhav**
**Is your Pull Request request related to another issue in this repository ?**      

This PR will resolve issue #33 .

**Describe what the PR does**    

Corrected typo in index.js file.

**State whether the PR is ready for review or whether it needs extra work**    

The PR is ready for review.

**Additional context**

### CODE REVIEW

File: src/components/index.js [hunk 1]
```diff
@@ -455,7 +455,7 @@ export default function SlateTranscriptEditor(props) {
   };
   return (
     <Container fluid style={{ backgroundColor: '#eee', height: '100vh', paddingTop: '1em' }}>
-      <style scoped>
+      <style scope>
```
Comment (pietrop) [inline src/components/index.js:458]:
```suggestion
      <style scoped>
```
I think it's with a `d`?

http://html5doctor.com/the-scoped-attribute/

Comment (pietrop) [reply src/components/index.js:458]: and line 466 was the one with the typo

Comment (96RadhikaJadhav) [reply src/components/index.js:458]: Oops! Apologies. Just a moment.

Comment (pietrop) [reply src/components/index.js:458]: no probl at all, and no rush 😄

Comment (96RadhikaJadhav) [reply src/components/index.js]: Done :)

**Comment by pietrop** [comment]
thanks @96RadhikaJadhav !

**Pull request merged**